ingestion_tools/scripts/data_validation/tests/annotation/test_metadata_annotation.py

```diff
@@ -0,0 +1,34 @@
+from typing import Dict
+
+import pytest
+from tests.test_deposition import HelperTestDeposition
+
+from common.fs import FileSystemApi
+
+
+@pytest.mark.annotation
+@pytest.mark.metadata
+@pytest.mark.parametrize("run_name, voxel_spacing", pytest.run_spacing_combinations, scope="session")
+class TestAnnotationMetadata:
+    def test_annotation_metadata(
+        self,
+        annotation_metadata: Dict[str, Dict],
+    ):
+        """Sanity check for annotation metadata and corresponding deposition metadata."""
+        for metadata in annotation_metadata.values():
+            assert metadata["deposition_id"]
+            assert isinstance(metadata["annotation_object"], dict)
+            assert isinstance(metadata["authors"], list)
+            assert isinstance(metadata["authors"][0], dict)
```

Comment:
Added, but it might be worth creating a schema & Pydantic for these metadata json files. Both useful as intellisense / insight for writing the ingestion code (as discussed with @Bento007 on slack) and also for validation of these json files